releasenotes/notes/oracle-sysmetrics-pga-tablespaces-9d39974aea78f760.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Oracle sysmetrics, pga process memory usage, tablespace usage with pluggable database (PDB) tags
```

Comment:
I haven't done a lot of writing of release notes so I don't have a good feeling for what the tone/style is but I wonder if this shouldn't be something like `Add oracle metrics ...`. 

Also not something blocking for this PR but I wonder if we need/should to have something like the [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/oracle/metadata.csv) that we have in the integrations to document the collected metrics. Maybe @DataDog/documentation has some ideas about that?

Reply:
@alai97 : Do you know if we have any mechanism (or plans for one) to list and describe the metrics emitted by core integrations in datadog-agent?

Reply:
Not one that I'm aware of, would you mind asking #websites on Slack and pointing to https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1169426657/Integrations#metadata.csv as an example?

Reply:
Added "Add" to the release notes.